modules/stats/drawdown/per_coin.py

```diff
@@ -49,3 +49,32 @@ def add_trade_fee(df, fee_percentage, trades_closed_opened):
     for opened, closed in trades_closed_opened:
         df.loc[opened, "profit_ratio"] *= fee_ratio
         df.loc[closed, "profit_ratio"] *= fee_ratio
+
+
+def find_trade_timestamps(closed_pair_trades):
```

Comment:
```suggestion
def get_trade_timestamps(closed_pair_trades):
```
get sounds more appropriate than find (as they are located already), I couldn't find what 'closed_pair_trades' entails exactly but 'pair' might not be needed? E.g. closed_trades would be sufficient?

Reply:
closed_pair_trades is used here because it's the list of closed trades, but only for one pair. In other parts of the code, closed_trades holds the list of all closed trades of all pairs combined, so I do feel that the name closed_pair_trades adds more information